api/v1alpha1/aerospikecluster_types.go

```diff
@@ -70,10 +70,39 @@ type AerospikeClusterSpec struct {
 	RackConfig RackConfig `json:"rackConfig,omitempty"`
 	// AerospikeNetworkPolicy specifies how clients and tools access the Aerospike cluster.
 	AerospikeNetworkPolicy AerospikeNetworkPolicy `json:"aerospikeNetworkPolicy,omitempty"`
+	// Certificates to connect to Aerospike. If omitted then certs are taken from the secret 'aerospike-secret'.
+	// +optional
+	OperatorClientCertSpec *OperatorClientCertSpec `json:"operatorClientCertSpec,omitempty"`
 	// Additional configuration for create Aerospike pods.
 	PodSpec AerospikePodSpec `json:"podSpec,omitempty"`
 }
 
+type OperatorClientCertSpec struct {
+	// This name will be added to tls-authenticate-client list by the operator
+	TLSClientName string `json:"tlsClientName"`
+	CertSource    `json:",inline"`
+}
+
+// CertSource represents the source of Aerospike ClientCert for Operator.
+// Only one of its members may be specified.
+type CertSource struct {
+	// +optional
+	SecretName *string `json:"secretName,omitempty"`
```

Comment:
secretName and namespace can be put in a struct like secretSource. CertSource should be like 
```
CertSource {
  *source1
  *source2
  *source3
}
```
In the future, we may also like to add some more fields inside any source. For e.g. I suggested the below schema for secret, not sure if you missed it or deliberately decided to skip it

```
type AerospikeSecretCertSource struct {
    SecretName string `json:"secretName"`
	CaCertsFilename    string `json:"caCertFilename"`
	// +optional
	ClientCertFilename string `json:"clientCertFilename"`
	// +optional
	ClientKeyFilename  string `json:"clientKeyFilename"`
}
```
ClientCertFilename, ClientKeyFilename are optional if tls-authenticate-client is "any"
https://docs.aerospike.com/docs/reference/configuration/#tls-authenticate-client